test/e2e/storage/persistent_volumes-local.go

```diff
@@ -307,6 +311,37 @@ var _ = SIGDescribe("PersistentVolumes-local [Feature:LocalPersistentVolumes] [S
 			Expect(err).NotTo(HaveOccurred())
 		})
 	})
+
+	Context("when using SELinux labels", func() {
+		It("a second pod using a different label should not be able to read or write to the same volume", func() {
```

Comment:
Oh I forgot, we don't actually have this enforcement today, so the test is actually checking the opposite, that the 2nd pod overwrites the label.  It seems like a strange/unintuitive behavior.  If folks agree that this is not the behavior that we want, then maybe we should not have it as a test case?

Or write a test case for the behavior we do want, and just keep it disabled until we fix the real issue.